apps/src/templates/sectionsRefresh/coteacherSettings/CoteacherSettings.jsx

```diff
@@ -77,7 +78,7 @@ export default function CoteacherSettings({
 
   return (
     <div className={styles.expandedSection}>
-      {i18n.coteacherAddInfo()}
+      {disabled ? i18n.coteacherLtiAddInfo() : i18n.coteacherAddInfo()}
```

Comment:
```suggestion
      {isLtiManagedSection ? i18n.coteacherLtiAddInfo() : i18n.coteacherAddInfo()}
```
Because `disabled` in this case changes the text to be lti specific, let's rename.